src/Modules/ChromiumExtraScriptTag.php

```diff
@@ -4,6 +4,7 @@
 
 namespace Gotenberg\Modules;
 
+/** @deperacted */
```

Comment:
deperacted -> deprecated

Reply:
Thanks @pieteroc! It has been fixed before the release though 😄

Reply:
Big 🥜  🧠  moment, yeah otherwise looks like a good changeset, appreciate it, thank you.